client/src/views/LevelUpView.tsx

```diff
@@ -30,6 +31,11 @@ const LevelUpView = () => {
   const { brute, updateBrute } = useBrute();
   const theme = useTheme();
 
+  const xpNeededForNextLevel = useMemo(
+    () => (brute ? getXPNeeded(brute.level + 1) : 0),
+    [brute],
+  );
```

Comment:
No need to memo this, as it's only used once in a callback.
It doesn't need to be declared